### PR TITLE
[t781] Some small fixes for media (flash)

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

File: templates/external-media.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            end: 2,
+            target: "Area2"
+          }
+        });
+
+      }
+
+      media.onReady( start );
+      
+      window.butter = butter;
```
Comment (jbuck) [inline templates/external-media.js]: delete it

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://butyr.org:8000/3d75b5ef564dd35/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://butyr.org:8000/3d75b5ef564dd35/output.txt

Total script time: 0.29 mins

Checking 72622a2ad9ef69d99c312960b05df9ac651f2b33 of pull request #264

#### Published

You can view your repo files at: http://butyr.org:8000/3d75b5ef564dd35

**Pull request merged**